### ISSUE TITLE
Keyboard-only user cannot scroll the Console

### ISSUE THREAD

**Issue by gtritchie** [rstudio/rstudio#7569]
### System details

    RStudio Edition : Desktop or Server
    RStudio Version : All versions up to latest 1.4 builds
    OS Version      : any
    R Version       : any

### Steps to reproduce the problem
- generate enough output in the RStudio Console that scrolling is required to see it all
- try to scroll back and see the earlier output without using a pointing device

### Describe the problem in detail
- cannot do it; must use the mouse

### Describe the behavior you expected
- keyboard-only users need a way to view the entire console output history
- NOTE: this is not a problem for screen-reader users, as they have the ability to focus on the output and read it (e.g. the Focus Console Output command)

See also: #6077,  #6518

- [x] I have read the guide to submitting good bug reports at https://github.com/rstudio/rstudio/wiki/Writing-Good-Bug-Reports . 
- [x] I have installed the latest version of RStudio and confirmed that the issue still persists.
- [x] If I am reporting a RStudio crash, I have included a diagnostics report. https://support.rstudio.com/hc/en-us/articles/200321257-Running-a-Diagnostics-Report
- [x] I have done my best to include a minimal, self-contained set of instructions for consistently reproducing the issue.

**Comment by gtritchie** [comment]
My mistake, this is not true. You can scroll via PgUp and PgDn keys. Would be nice to also scroll via arrow keys, but those are used for console history. Closing this as, uh, "by design".
